easy-ui-react/src/styles/global.scss

```diff
@@ -22,3 +22,8 @@ select,
 textarea {
   font-family: inherit;
 }
+
+button,
+a {
+  cursor: pointer;
+}
```

Comment:
does this make sense as a global style?

Reply:
i would probably try to isolate css changes to our components where possible, only because global styles can conflict with external libraries and such. if we want EZUI to work anywhere, keeping it encapsulated per component would be best